brave/browser-laptop
# Fixes sortTable default sort

Resolves #13721

## Submitter Checklist:


## Test Plan:
1. start browser on clean profile
2. enable payments
3. close browser
4. run npm run add-simulated-synopsis-visits 500 or 1k
5. start browser again and make sure that table is sorted by % desc
6. make sure that there is no CPU spikes and you can click on buttons when on payments page

## Reviewer Checklist:


Tests
